app/services/duplicate_benchmark_resolver.rb

```diff
@@ -34,13 +37,17 @@ def benchmarks
     end
 
     def migrate_rules(existing_bm, duplicate_bm)
+      logger.info(
```

Comment:
All of these break rubocop - since this is temporary code only meant to run on a specific migration, and it's just adding logger code, would you be OK with disabling rubocop here?